Node/samples/app.js

```diff
@@ -145,13 +149,29 @@ bot.dialog('StartNewReplyChain', function (session) {
 });
 bot.dialog('MentionUser', function (session) {
     // user name/user id
-    var toMention = {
-        name: 'Bill Zeng',
-        id: userId
+    var user = {
+        id: userId,
+        name: 'Bill Zeng'
     };
-    var msg = new teams.TeamsMessage(session).text(teams.TeamsMessage.getTenantId(session.message));
-    var mentionedMsg = msg.addMentionToText(toMention);
-    session.send(mentionedMsg);
+    var mention = new teams.UserMention(user);
+    var msg = new teams.TeamsMessage(session).addEntity(mention).text(mention.text + ' ' + teams.TeamsMessage.getTenantId(session.message));
+    session.send(msg);
+    session.endDialog();
+});
+bot.dialog('MentionChannel', function (session) {
+    // user name/user id
+    var channelId = null;
+    if (session.message.address.conversation.id) {
+        var splitted = session.message.address.conversation.id.split(';', 1);
+        channelId = splitted[0];
+    }
+    var channel = {
+        id: channelId,
+        name: 'All'
```

Comment:
For the example, I think it would be more helpful to show how to retrieve the real name of channel (through Get channels API), instead of just putting hardcoded name.

Reply:
Jingjing please ignore auto generated file